packages/env-tests/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "@celo/contractkit": "0.4.18",
+    "@celo/contractkit": "0.4.19-dev",
```

Comment:
was this a published version rather than dev intentionally? will be need to change it back after new contractkit version is published?

Reply:
@nambrot can you help us here? Should it be non-dev by intention?

Reply:
Ok, I've reverted the change, so `env-tests` still depends on the released `0.4.18`

Reply:
Yeah I was trying to use a published version intentionally to make moving out of the monorepo easier